### PR TITLE
Initial updates to the readme

### LINKED ISSUE #43: Improve the README.md
The current README.md has some basic info and it is mixed with CONTRIBUTING.md.

It should be split to contain a banner, badges, clearer explanation of the applications in the repository and how to navigate them etc.

### PR DESCRIPTION
**Pull request by Braqzen**
Closes #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
+    <a href="https://github.com/FuelLabs/sway-applications/actions/workflows/ci.yml" alt="CI">
+        <img src="https://github.com/FuelLabs/sway-applications/actions/workflows/ci.yml/badge.svg" />
+    </a>
+    <a href="https://crates.io/crates/forc/0.13.2" alt="forc">
+        <img src="https://img.shields.io/badge/forc-v0.13.2-orange" />
+    </a>
+    <a href="https://discord.gg/xfpK4Pe">
+        <img src="https://img.shields.io/discord/732892373507375164?color=blue&logo=discord&logoColor=ffffff&labelColor=6A7EC2&label=Discord" />
+    </a>
+</p>
```
Comment (adlerjohn) [inline README.md:15]: Avoid raw HTML. Use markdown for including images. See https://github.com/FuelLabs/fuels-ts/blob/master/README.md?plain=1 for reference.

Comment (Braqzen) [reply README.md:15]: I did initially use markdown however that does not center align the badges and logo. It looks better that way. What is the reason for not using HTML?

**Comment by Braqzen** [comment]
huh, the CI did not run again

File: README.md (hunk 1 above)
Comment (adlerjohn) [reply README.md:15]: Ah I see. Can you just wrap the markdown including the badges and image in a `<p align="center"> ... </p>` block, or does that not work?

Comment (Braqzen) [reply README.md:15]: I will try it and get back to you.

Comment (Braqzen) [reply README.md:15]: Nope, markdown does not support positioning. [Ref](https://stackoverflow.com/a/12091563)

We can either use HTML to align or we do not position in the center

Comment (adlerjohn) [reply README.md:15]: Up to you, don't feel strongly about either direction.

File: forc_badge_check.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CI_VERSION=`grep "FORC_VERSION:" .github/workflows/ci.yml | cut -d " " -f4`
+BADGE_LINK_VERSION=`grep "https://crates.io/crates/forc/" ./README.md | cut -d "/" -f6 | cut -d "\"" -f1`
+BADGE_VERSION=`grep "https://img.shields.io/badge/forc-v" ./README.md | cut -d "/" -f5 | cut -d "-" -f2  | cut -d "v" -f2`
```
Comment (adlerjohn) [inline forc_badge_check.sh:5]:
```suggestion
CI_VERSION="$(grep "FORC_VERSION:" .github/workflows/ci.yml | cut -d " " -f4)"
BADGE_LINK_VERSION="$(grep "https://crates.io/crates/forc/" ./README.md | cut -d "/" -f6 | cut -d "\"" -f1)"
BADGE_VERSION="$(grep "https://img.shields.io/badge/forc-v" ./README.md | cut -d "/" -f5 | cut -d "-" -f2  | cut -d "v" -f2)"
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,34 @@
...
+
+## Overview
 
 The purpose of this repository is to contain end-to-end applications that are written in Sway in order to demonstrate what can be built.
 
-This means that generally a project will consist of a Sway contract and a user interface in order to interact with the contract however that is not a hard rule.
+This means that a project will generally consist of a Sway contract and a user interface in order to interact with the contract however that is not a hard rule.
+
+> **NOTE:** Sway is a language under heavy development therefore the applications may not be the most ergonomic. Over time they should receive updates / improvements in order to demonstrate how Sway can be used in real use cases.
```
Comment (adlerjohn) [inline README.md:23]: https://github.com/github/feedback/discussions/16925
```suggestion
> **Note**
> Sway is a language under heavy development therefore the applications may not be the most ergonomic. Over time they should receive updates / improvements in order to demonstrate how Sway can be used in real use cases.
```

**Review by matt-user — APPROVED** [review]

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**